eox_nelp/user_profile/required_fields_validation.py

```diff
@@ -139,7 +172,7 @@ def validate_account_fields(user, account_fields):
     Returns:
         dict: A dictionary with invalid account fields and their errors.
     """
-    return validate_user_fields(user, account_fields)
+    return validate_user_fields(user, user, account_fields)
```

Comment:
```suggestion
    return validate_user_fields(user=user, instance=user, fields=account_fields)
```
Seems a little better =)